### ISSUE TITLE
parallelMesh 

### ISSUE THREAD

**Issue by wrxhardworking** [SalomePlatform/smesh#2]
Is this a recent development? I didn't see this class in the previous version

**Comment by mohd-afeef-badri** [comment]
Yes this is indeed one of the new functionalities introduced in SALOME 9.12 (still under development for other features).  The idea is to use coarse-grained approach for parallelization with shared memory paradigm (multi-thread), i.e, several volumes can me meshed across threads. Please see this for further details https://docs.salome-platform.org/latest/gui/SMESH/parallel_compute.html

**Comment by mohd-afeef-badri** [comment]
@wrxhardworking closing this issue. Do not hesitate to reopen incase you want more info.
